RubeesEat.UnitTests/Model/TestDomain.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Immutable;
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+public static class TestDomain
+{
+    public static readonly List<Person> TestPersons =
+    [
+        Person.Create("DefaultUserFirstName", "DefaultUserLastName"), 
+        Person.Create("SecondUserFirstName", "SecondUserLastName"),
+        Person.Create("ThirdUserFirstName", "ThirdUserLastName"),
+    ];
+
+    public static readonly List<Bill> TestBill = [];
```

Comment:
Name `TestBill` is a bit strange for a list. I would suggest `AllBills`. Please also add properties for the individual bill objects to allow their use in the test without indexer. You can also give them names like `BillFladerei` or something to differentiate them better in the tests.